src/Core/Managed/Net45/Extensibility/Implementation/RichPayloadEventSource.cs

```diff
@@ -78,6 +83,40 @@ public void Process(ITelemetry item)
         }
 
         /// <summary>
+        /// Record an operation start.
+        /// </summary>
+        /// <param name="operation">The opeation which is about to start.</param>
+        public void ProcessOperationStart(OperationTelemetry operation)
+        {
+            if (this.EventSourceInternal == null)
+            {
+                return;
+            }
+
+            if (this.EventSourceInternal.IsEnabled(EventLevel.Informational, Keywords.Operations))
+            {
+                operationStartStopHandler(operation, EventOpcode.Start);
+            }
+        }
+
+        /// <summary>
+        /// Record an operation stop.
+        /// </summary>
+        /// <param name="operation">The operation which has just stopped.</param>
+        public void ProcessOperationStop(OperationTelemetry operation)
```

Comment:
I think I see what you're getting at: Could we pair "Start" with "Track" instead of "Start"/"Stop".
The important thing is to use the Start and Stop opcodes so that the TraceEvent library can pair them up. (It _could_ be done with Start+Track, but it requires a custom processor).
So, Stop is designed to be paired with Start; hence a different keyword.
Order is significant (Start has to be before Stop).
I do see scenarios where we listen only to "Operations" and not general telemetry keywords.